service/http.balancer/pom.xml

```diff
@@ -98,6 +98,24 @@
             <groupId>org.kaazing</groupId>
             <artifactId>k3po.junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
just fyi the gateway.transport.wsn  pom.xml includes some of these and they are not under the plugin and the versions are hard coded:

<dependency>
            <groupId>org.kaazing</groupId>
            <artifactId>specification.ws</artifactId>
            <version>2.0.1</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.kaazing</groupId>
            <artifactId>specification.x-kaazing-idle-timeout</artifactId>
            <version>2.2.0</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.kaazing</groupId>
            <artifactId>specification.x-kaazing-ping-pong</artifactId>
            <version>2.2.0</version>
            <scope>test</scope>
        </dependency>